Neos.ContentRepository.Core/Classes/Projection/ProjectionInterface.php

```diff
@@ -26,9 +26,9 @@ interface ProjectionInterface
     public function setUp(): void;
 
     /**
-     * Determines the status of the projection (not to confuse with {@see getState()})
+     * Determines the setup status of the projection
      */
-    public function status(): ProjectionStatus;
+    public function setUpStatus(): ProjectionSetupStatus;
```

Comment:
revert change to `ProjectionStatus` and just add in comment that it is the setup state, which migrations need to be run